### PR TITLE
Update webpack browsers list

### PR DESCRIPTION
**Pull request by noahbenham**
### Summary

~This defines a default Browsers List, which is used automatically by Autoprefixer per [best practices](https://github.com/postcss/autoprefixer/#options).~

~We're not currently consuming terra-toolkit. Could someone please verify this is working?~

This updates the browsers list in `webpack.config.js` from [this set](http://browserl.ist/?q=ie+%3E%3D+10%2C+last+2+versions%2C+last+2+android+versions%2C+last+2+and_chr+versions%2C+iOS+%3E%3D+10) to [a new one](http://browserl.ist/?q=iOS+%3E%3D+10%2C+last+2+and_chr+versions%2C+last+2+android+versions%2C+last+2+chrome+versions%2C+last+2+edge+versions%2C+last+2+firefox+versions%2C+last+2+ie+versions%2C+last+2+safari+versions). Keep in mind this reduces coverage from 86.74% to 74.04%.

### Additional Details

@cerner/terra

[CONTRIBUTORS.md]: ../blob/master/CONTRIBUTORS.md
[License]: ../blob/master/LICENSE

### CODE REVIEW

File: .browserslistrc [hunk 1]
```diff
@@ -0,0 +1,5 @@
+ie >= 10
+last 2 versions
```
Comment (alex-bezek) [inline .browserslistrc]: careful with `last 2 versions` these are and clauses. Compare these 2
http://browserl.ist/?q=ie+%3E%3D+10%2C+last+2+android+versions%2C+last+2+and_chr+versions%2C+iOS+%3E%3D+10
vs
http://browserl.ist/?q=ie+%3E%3D+10%2C+last+2+android+versions%2C+last+2+and_chr+versions%2C+iOS+%3E%3D+10%2C+last+2+versions

last 2 versions gets you awesome browsers like Blackberry 7

https://jamie.build/last-2-versions

**Comment by emilyrohrbough** [comment]
@noahbenham To verify you will need to create a branch in terra-dev-site that updates terra-toolkit to point to this branch. Then, from there, you will need create a 'do not merge' PR in one of the main repos (clinical would be fastest) that updates terra-dev-site and terra-toolkit to use the respective branches. We need to specify both to ensure the same version of toolkit is pulled in.

File: .browserslistrc (hunk 1 above)
Comment (mjhenkes) [reply .browserslistrc]: Thanks for the info @alex-bezek, I wouldn't have considered that. Nice test site too.

Comment (noahbenham) [reply .browserslistrc]: Yikes, so this has been a problem for awhile.

File: .browserslistrc [hunk 2]
```diff
@@ -0,0 +1,4 @@
+ie >= 10
+last 2 android versions
+last 2 and_chr versions
+iOS >= 10
```
Comment (mjhenkes) [inline .browserslistrc]: Add:
last 2 edge versions
last 2 chrome versions
last 2 firefox versions

Comment (noahbenham) [reply .browserslistrc]: 797bc809f79f4c3b93205d0b4657679c42a42011

http://browserl.ist/?q=ie+%3E%3D+10%2C+last+2+android+versions%2C+last+2+and_chr+versions%2C+last+2+edge+versions%2C+last+2+chrome+versions%2C+last+2+firefox+versions%2C+iOS+%3E%3D+10

**Comment by noahbenham** [comment]
Created [Terra Clinical PR for testing](https://github.com/cerner/terra-clinical/pull/334/) per @emilyrohrbough.

File: .browserslistrc [hunk 3]
```diff
@@ -0,0 +1,7 @@
+ie >= 10
```
Comment (bjankord) [inline .browserslistrc]: nit: change this to `last 2 ie versions`.

File: .browserslistrc [hunk 4]
```diff
@@ -0,0 +1,7 @@
...
+last 2 android versions
+last 2 and_chr versions
+last 2 edge versions
+last 2 chrome versions
+last 2 firefox versions
```
Comment (bjankord) [inline .browserslistrc]: Also need to add `last 2 safari versions`

**Comment by emilyrohrbough** [comment]
Noah, have you read the notes on Shareable Browserlist config? https://github.com/browserslist/browserslist#shareable-configs

Can you verify the browsers provided in the browserlist are being used in terra-clinical? If no config is provided, the default will be `> 0.5%, last 2 versions, Firefox ESR, not dead`

**Review by emilyrohrbough — APPROVED** [review]

**Comment by emilyrohrbough** [comment]
@noahbenham and I did some debugging and determined when using `.browserslistrc` projects pulling in terra-toolkit were not receiving the defined browsers, but was using the browsers list default values. Given the intention of the default webpack config we decided it would be more appropriate to update the defined browserslist value within the webpack config.

**Review by mjhenkes — APPROVED** [review]
Thanks for taking a look @noahbenham

**Comment by noahbenham** [comment]
I've updated the PR summary with the list of browsers before & after this change. Browser coverage is slightly reduced, let me know if this list needs further tweaks.

**Comment by mjhenkes** [comment]
We're going to close this PR in favor of the terra-toolkit V4 changes emily is making.

**Pull request closed without merging**